module.rb

```diff
@@ -0,0 +1,7 @@
+module FunBehaviors
```

Comment:
This module breakout is good and I like the naming - it's not just a module for the `HighFive` behavior but could potentially handle other `FunBehaviors` in the future. I recommend changing the name of the file to reflect the behavior of the module - from `module.rb` to `fun_behaviors.rb`.